test/unit/lib/dispatcher.js

```diff
@@ -52,6 +52,7 @@ describe('Dispatcher', () => {
             let spyA = sinon.spy();
             let spyB = sinon.spy();
             let tokenA = Dispatcher.register(spyA);
+            /* eslint-disable no-unused-vars */
```

Comment:
I'm assuming it was added so the linter would ignore the unused `tokenB` variable.

I would probably just remove the variable, changing the line from `let tokenB = Dispatcher.register(spyB);` to `Dispatcher.register(spyB);`.